src/main.cpp

```diff
@@ -569,6 +616,9 @@ auto main(int _argc, char* _argv[]) -> int
         const auto eviction_check_interval = http_server_config.at(json::json_pointer{"/authentication/eviction_check_interval_in_seconds"}).get<int>();
         bearer_token_eviction_manager eviction_mgr{ioc, std::chrono::seconds{eviction_check_interval}};
 
+		// TODO: Don't lean on BT timer
```

Comment:
What does BT stand for?

Reply:
BT = bearer token. Was referring to the `bearer_token_eviction_manager`'s `eviciton_check_interval` time.

Reply:
Got it.

Reply:
Done